src/main/java/sg/ncl/testbed_interface/TeamPageApplyTeamForm.java

```diff
@@ -23,6 +23,16 @@
     
     @NotNull(message="Please select one of them")
     private String isPublic = TeamVisibility.PUBLIC.toString();
+
+    public String getJoinTeamReason() {
+        return joinTeamReason;
+    }
+
+    public void setJoinTeamReason(String joinTeamReason) {
+        this.joinTeamReason = joinTeamReason;
+    }
+
+    private String joinTeamReason;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)